runtime/tools/python/ttrt/common/api.py

```diff
@@ -841,6 +841,8 @@ def execute(self):
             def _execute(binaries):
                 import ttrt.runtime
                 import torch
+                # Don't summarize printing tensors.
+                torch.set_printoptions(precision=3, threshold=10000, linewidth=200)
```

Comment:
Are these numbers for threshold and linewidth good enough for any tensor size? I would imagine there is a way to set print options to print full tensor in a more generic way, without these hardcoded numbers